js/viz/vector_map/control_bar.js

```diff
@@ -225,6 +247,15 @@ ControlBar.prototype = {
         that._isActive = that._isEnabled && that._flags && that._params.projection.isInvertible();
         if(that._isActive) {
             that._root.linkAppend();
+
+            if(!that._isPanVisible) {
+                that._panControl.remove();
+                that._trackersPan.remove();
+            }
+            if(!that._isZoomVisible) {
+                that._zoomBar.remove();
+                that._trackersZoom.remove();
```

Comment:
After removing this groups, what will be if the user set visibility to true after?